src/Features/Core/Portable/DocumentationComments/AbstractDocumentationCommentFormattingService.cs

```diff
@@ -155,17 +175,18 @@ private static void AppendTextFromNode(FormatterState state, XNode node, Compila
             {
                 foreach (var attribute in element.Attributes())
                 {
-                    AppendTextFromAttribute(state, element, attribute, attributeNameToParse: "cref");
+                    AppendTextFromAttribute(state, element, attribute, attributeNameToParse: "cref", SymbolDisplayPartKind.Text);
                 }
 
                 return;
             }
             else if (name == "paramref" ||
                      name == "typeparamref")
             {
+                var kind = name == "paramref" ? SymbolDisplayPartKind.ParameterName : SymbolDisplayPartKind.TypeParameterName;
                 foreach (var attribute in element.Attributes())
                 {
-                    AppendTextFromAttribute(state, element, attribute, attributeNameToParse: "name");
+                    AppendTextFromAttribute(state, element, attribute, attributeNameToParse: "name", kind);
```

Comment:
i would have expected to just see this.

Reply:
The last commit is close to just this. 👍